PHPGraphQLArticle.md

```diff
@@ -212,55 +202,49 @@ query UserPosts{
+I was then able to come up with a simple way to structure the app. This was huge benefit as I was looking for a GraphQL server whose sole purpose was to serve frontend - but I found much more. A good code structure that is easily testable and very well documented. 
 
-And that was a fantastic bonus to me. I was looking for a GraphQL server with the only purpose to serve the frontend, but found a lot more. A good code structure, easily testable, and very well documented by default. 
+Lets continue with our exploration of the architecture. I'll guide you through the App in the same order I would when developing a new feature.
 
-Talking about tests, another decision taken was to run most tests over the API layer, what also proved to be a very good decision. In fact, my working proccess of a new feature now allways starts from the schema going to the tests on the top of that schema. Those are the beginning step prior to actual implementation of the features.  
-
-To make our exploration of the architecture, I'll guide you through the App in the same order I use to develop a new feature. So that you can understand the app and also the proccess that is working fine for me. 
-
-After we see a complete development cycle we will also take a look at some technical details. In special those required to integrate the libs we are using together. 
+Once we have covered a complete development cycle we will also take a look at some technical details. 
 
 # Hands On
 
 ### The App We are Going to Work With
 
-The view is usually the easiest part to understand an application domain. So let's look at it before we install the backend and look at it's API. 
-
-We are gonna work on a Knowledge Management App. The main purpose of this app is to organize knowledge that is shared through Telegram and other channels in the future. 
+We are going to work on a Knowledge Management App. The main purpose of this app is to organize knowledge that is shared through Telegram. 
 
-The mais screen, as shown below, is where you organize messages in specific threads or conversations and put tags on those threads. 
+The main screen, as shown below, is where we organize messages, in specific threads or conversations, and put tags on those threads. 
 
-> In this animation, the user is selecting 9 messages and creating a thread. After that, he goes to the created thread and add a tag to classify that thread as a SQL thread. 
+> In this animation, the user is selecting messages, creating a thread and adding a tag.
 
 ![Create a Thread and Tag it!](./images/createThread.gif)
 
-We are going to work on a very simple example because our goal here is to explain about our architecture, but this App can give you interesting data structures to serve as an example to deeper study:
+We are going to cover a very simple example because our goal here is simply to explain the architecture.If you are interest in diving deeperm this App can give you some other data structures to study:
```

Comment:
We are going to cover a very simple example because our goal here is purely to explain the architecture. If you are interested in diving deeper, this App can give you some other data structures to study: